catalyst-gateway/bin/src/db/index/block/mod.rs

```diff
@@ -0,0 +1,95 @@
+//! Index a block
+//! Primary Data Indexing - Upsert operations
+
+pub(crate) mod certs;
+pub(crate) mod cip36;
+pub(crate) mod txi;
+pub(crate) mod txo;
+
+use cardano_chain_follower::MultiEraBlock;
+use certs::CertInsertQuery;
+use cip36::Cip36InsertQuery;
+use tracing::{debug, error};
+use txi::TxiInsertQuery;
+use txo::TxoInsertQuery;
+
+use super::{queries::FallibleQueryTasks, session::CassandraSession};
+use crate::service::utilities::convert::i16_from_saturating;
+
+/// Add all data needed from the block into the indexes.
+#[allow(clippy::similar_names)]
```

Comment:
I think it's worth to fix this clippy lint
```suggestion
```

Reply:
Yes, looks like a hangover from a refactor.  Just testing it now.

Reply:
Done